config/community_images.json

```diff
@@ -17,4 +17,14 @@
     "image": "us.gcr.io/anvil-gcr-public/anvil-rstudio-bioconductor:3.13.2",
     "requiresSpark": false,
     "isRStudio": true
+},
+{
+    "id": "OpenVINO",
+    "label": "OpenVINO (OpenVINO 0.5.0, Python 3.7.10, GATK 4.2.0.0)",
```

Comment:
@ravi9 let me know if you're OK with this label.

Reply:
(This will be the label displayed in the UI drop-down)

Reply:
Hi @rtitle,  Could you update with the following?
 ```
 "id": "OpenVINO integration with Tensorflow",
  "label": "OpenVINO integration with Tensorflow (openvino-tensorflow 0.5.0, Python 3.7.10, GATK 4.2.0.0)",
```

Reply:
Pushed